pyproject.toml

```diff
@@ -8,7 +8,7 @@ requires = [
     "cffi>=1.15.1",
     "typing-extensions>=4.7.0",
     "future>=0.18.3",
-    "tensorrt-cu12>=10.6.0,<10.8.0",
+    "tensorrt-cu12==10.7.0",
```

Comment:
Do multiple TRT versions get tested now in CI with this change ?

Reply:
It should right? Also 10.6 should be supported but we need at least 10.6 because of the QDP feature

Reply:
Do you mean we need to keep `tensorrt-cu12>=10.6.0,<10.8.0`?

Reply:
yeah